Sources/OpenAI/Client.swift

```diff
@@ -827,6 +830,40 @@ public final class Client {
                 }
             }
     }
+    
+    // MARK: Content Filter
+    /**
```

Comment:
Nitpick: Blank line after demarcation

```suggestion
    // MARK: Content Filter

    /**
```